README.md

```diff
@@ -1,16 +1,137 @@
 # Helium-Java
 
+# Build a Docker containerized, secure Java Web API application using Managed Identity, Key Vault, and Cosmos DB that is designed to be deployed to Azure App Service or AKS
```

Comment:
This renders REALLY LARGE and is also a lint error. You should be able to remove the # or change it to ## or ###

Reply:
ok took out the > and put in ### and it looks more reasonable.